### ISSUE TITLE
Utilize autoloading

### ISSUE THREAD

**Issue by jhedstrom** [chargebee/chargebee-php#27]
The `lib/ChargeBee.php` file currently loads all of this library's code. It would be nice to adopt autoloading as provided by composer. This would be easier to do after #23 is done.

**Comment by karololszacki** [comment]
This is a duplicate of #11... but it's super annoying.
Maybe we should all create issues about this to shed some light on it...

**Comment by robbieaverill** [comment]
Just popping in to say hello! We've noticed that the manual "autoloading" of files in this repository is contributing to a decent amount of our startup time. PSR-4 would be great. If anyone is maintaining this, I'd be happy to make a PR for it. Cheers.

**Comment by cb-rakesh** [comment]
This has been fixed in v3.0.0
